src/makeRobot.js

```diff
@@ -38,7 +38,63 @@
  * @return {object}
  */
 function makeRobot(name, wheels, version) {
-  // write code here
+  const robot = {
+    name: name,
+    chipVersion: version,
+    wheels: wheels,
```

Comment:
https://mateacademy.slack.com/archives/G01KEC0048K/p1615292056066700
```suggestion
    name,
    chipVersion: version,
    wheels,
```